README.md

```diff
@@ -111,6 +111,41 @@ module.exports = (argv) => {
 };
 ```
 
+## Http proxy
+
+#### Enabling proxy
+
+The server contains a simple http proxy. 
+The proxy must be configured in the config file.
+To enable this proxy:
+
+```sh
+angular-http-server --config configs/angular-http-server.config.js --useProxy true
+```
+
+#### configuring proxy
+
+To configure the proxy add a proxyConfig object to your config file. 
+The proxyConfig should be an array of configs with two properties: a forward property which must be a string array listing url parts which should trigger the proxy, and a target property which should define the target to proxy to.
+
+simple example:
+
+```js
+module.exports = {
+    proxyConfig: [
```

Comment:
as we are in a config file anyway, how about just calling this `proxy`

Reply:
done